base_ubl/models/ubl.py

```diff
@@ -18,7 +21,7 @@
     logger.debug('Cannot import PyPDF2')
 
 
-class BaseUbl(models.AbstractModel):
+class BaseUbl(models.TransientModel):
     _name = 'base.ubl'
```

Comment:
TransientModel is a class that store data in the database temporary. and also use when create wizard. AbstractModel class also perform same functionality.
* If the _name as the same value as the inherited class it will do a basic inheritance.
* If the class _inherit one model and  set a  _name different, it will create a new model in a new database table.
* we can do using both class